backend/pkg/connect/config.go

```diff
@@ -12,17 +12,30 @@ package connect
 import (
 	"flag"
 	"fmt"
+	"time"
+)
+
+const (
+	defaultConnectTimeout = 15 * time.Second
+	defaultReadTimeout    = 60 * time.Second
+	defaultRequestTimeout = 6 * time.Second
 )
 
 type Config struct {
-	Enabled  bool            `yaml:"enabled"`
-	Clusters []ConfigCluster `yaml:"clusters"`
+	ConnectTimeout time.Duration   `yaml:"connectTimeout"` // used for connectivity test
+	ReadTimeout    time.Duration   `yaml:"readTimeout"`    // overall REST/HTTP read timeout
+	RequestTimeout time.Duration   `yaml:"requestTimeout"` // timeout for REST requests to Kafka Connect
```

Comment:
Makes sense to me, could you move them below the existing properties in that config struct (as they are more likely to be configured)

Reply:
OK, no problem